Resources/public/js/views/tabs/ez-locationviewdetailstabview.js

```diff
@@ -34,7 +60,12 @@ YUI.add('ez-locationviewdetailstabview', function (Y) {
                 currentVersion = content.get('currentVersion'),
                 translationsList = currentVersion.getTranslationsList(),
                 creator = null,
-                owner = null;
+                owner = null,
+                that = this,
+                sortFieldIdentifiers = this._getOrderingMethods(),
+                sortFields = Y.Array.map(sortFieldIdentifiers, function(id) {
```

Comment:
you should put that in another method (render is quite big already). Also, Y.Array.map accepts the context in which the function should be executed in its third argument, you can get rid of the `that`.